abce/inventory.py

```diff
@@ -3,14 +3,18 @@
 from abce.notenoughgoods import NotEnoughGoods
 from .expiringgood import ExpiringGood
 
-
 epsilon = get_epsilon()
 
 
+def isclose(a, b):
```

Comment:
Yeah this is how it is implemented at elsewhere in the code.

Reply:
what do you mean